Application/EdFi.Ods.Api/Helpers/AssemblyLoaderHelper.cs

```diff
@@ -13,18 +13,26 @@
 using System.Runtime.Loader;
 using EdFi.Common.Extensions;
 using EdFi.Ods.Api.Constants;
+using EdFi.Ods.Api.Conventions;
+using EdFi.Ods.Api.Models;
+using EdFi.Ods.Api.Providers;
+using EdFi.Ods.Common.Conventions;
 using EdFi.Ods.Common.Extensibility;
+using EdFi.Ods.Common.Models;
 using EdFi.Ods.Common.Models.Validation;
 using EdFi.Ods.Common.Validation;
 using FluentValidation;
 using log4net;
+using log4net.Repository.Hierarchy;
+using Newtonsoft.Json;
 
 namespace EdFi.Ods.Api.Helpers
 {
     public static class AssemblyLoaderHelper
     {
         private static readonly ILog _logger = LogManager.GetLogger(typeof(AssemblyLoaderHelper));
-
+        private const string AssemblyMetadataSearchString = "assemblyMetadata.json";
+        private static string _jsonFile ;

```

Comment:
You've got a space before the final `;`, but more importantly, why is this a static variable?

This is potentially dangerous as it means there is 1 copy of this variable for all threads in the application to share. In the event multiple calls were made in parallel to this class (which in this case I don't think will actually happen), the threads would be overwriting each others values. It's just not a good practice.

Also, this value is only used inside of one method. Please make it a local variable inside of that method.